### PR TITLE
2015 site. 

### PR DESCRIPTION
**Pull request by hunterowens**

### CODE REVIEW

File: 2014/CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+juliacon.org
```
Comment (StefanKarpinski) [inline 2014/CNAME]: The `CNAME` file should stay at the top level.

Comment (hunterowens) [inline 2014/CNAME]: Derp. My bad, fixing.

**Comment by jiahao** [comment]
I've merged the updates and put in some basic links to the CFP and the proposal submission form. This should be good enough for a basic 2015 site.

**Comment by StefanKarpinski** [comment]
Cool. Can we do this so that the top level URL is a temporary redirect (HTTP 302) to juliacon.org/2015/? That way when people save links, they will continue to work and next year will just have different links. Would also be good to mention the year in the front page since otherwise this looks a lot like last year's.

**Pull request merged**